apps/emqx_prometheus/src/emqx_prometheus.erl

```diff
@@ -1118,6 +1157,13 @@ collect_stats_json_data(StatsData, StatsClData) ->
 collect_cert_json_data(Data) ->
     collect_json_data_(Data).
 
+collect_client_json_data(Data0) ->
+    ShutdownCounts = maps:with([emqx_client_disconnected_reason], Data0),
+    Data = maps:without([emqx_client_disconnected_reason], Data0),
+    JSON0 = collect_json_data(Data),
+    JSON1 = collect_json_data_(ShutdownCounts),
+    lists:flatten([JSON0 | JSON1]).
```

Comment:
Q: Why is "shutdown counts" metrics accumulated (or rather injected?) slightly differently than other client metrics?

Reply:
Because the current logic in this module cannot handle metrics with more than one label...  The JSON structure is not quite amenable to that.  :melting_face: 

_At most_ it may have the `node` label, nothing more.